drivers/hmis/lib/form_data/records/client.json

```diff
@@ -181,6 +191,29 @@
           "pick_list_reference": "NoYesReasonsForMissingData"
         }
       ]
+    },
+    {
+      "type": "GROUP",
+      "required": false,
+      "link_id": "image",
+      "text": "Image",
+      "enable_behavior": "ANY",
+      "enable_when": [
+        {
+          "question": "client-id",
+          "operator": "EXISTS",
+          "answer_boolean": false
+        }
+      ],
+      "item": [
+        {
+          "type": "IMAGE",
+          "required": false,
+          "link_id": "image_blob_id",
+          "text": "Image",
+          "field_name": "imageBlobId"
+        }
+      ]
```

Comment:
I think this should come out.  There's a section on this form that is empty and shouldn't be there.

Reply:
That's not what I'm seeing. It looks good to me -- the section appears on the New client form, but not on the Update client form.

There was an issue with the `Image` header incorrectly appearing on the left nav, but that should be fixed with https://github.com/greenriver/hmis-frontend/pull/117